diff --git a/node_modules/@msg91comm/react-native-sendotp/src/OTPVerification.tsx b/node_modules/@msg91comm/react-native-sendotp/src/OTPVerification.tsx
index ae4e51c..01e011f 100644
--- a/node_modules/@msg91comm/react-native-sendotp/src/OTPVerification.tsx
+++ b/node_modules/@msg91comm/react-native-sendotp/src/OTPVerification.tsx
@@ -1,6 +1,6 @@
 import { WebView } from 'react-native-webview';
 import React from 'react'
-
+// Patched to force newer react-native-webview version
 interface Props {
   onVisible: boolean,
   authToken: string,
